### ISSUE TITLE
Disable too-few-public-methods for attrs.

### ISSUE THREAD

**Issue by Beefy-Swain** [PyCQA/pylint#2988]
### Is your feature request related to a problem? Please describe
Dataclasses, and some similar types of classes disable the `too-few-public-methods` check, but attrs does not. In addition there is not any way to specify types of classes to automatically ignored for those types of checks.

### Describe the solution you'd like
Attrs should probably by default disable `too-few-public-methods`.

Another potential option is to allow the user to specify decorators in the pylintcr that should disable certain checks.

### Additional context
I def could just be dumb and unaware of the solution to this problem, but I did google around a lot, and read through `design_analysis.py` to no avail.

**Comment by PCManticore** [comment]
Hey @Beefy-Swain Thanks for the report, this now should be fixed in `master`.

**Comment by xqgex** [comment]
The error still exists with pylint version 2.17.0 and attrs version 23.1.0

Code:

```Python
""" Module description. """

from attrs import define, field, frozen
from attrs.validators import instance_of


@define
class Bar:
    """ Class description. """
    field: int = field(validator=instance_of(int))


@frozen
class Foo:
    """ Class description. """
    field: int = field(validator=instance_of(int))


print(Bar(5))
print(Foo(7))
```

Output:

```
************* Module test
test.py:8:0: R0903: Too few public methods (0/2) (too-few-public-methods)
test.py:14:0: R0903: Too few public methods (0/2) (too-few-public-methods)

------------------------------------------------------------------
Your code has been rated at 7.50/10
```
